submissions/elijahZgambo/database/init.sql

```diff
@@ -0,0 +1,53 @@
+/* Creating food_bag database */
+create database food_bag;
+
+use food_bag;
+/* Creating food table */
+create table food ( 
+	id int not null auto_increment, 
+	type varchar(20) not null,
```

Comment:
There are four different food categories on the homepage that would be added to this table. All these columns are required. So change line 8 to :  type ENUM('fast_food', 'vegetable_fruits', 'drinks_cocktails', 'restaurants') NOT NULL DEFAULT 'fast_food',